client/go/outline/vpn/method_channel.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vpn
+
+import "github.com/Jigsaw-Code/outline-apps/client/go/outline"
+
+func RegisterMethodHandlers() {
```

Comment:
See my comment in outline/method_channel.go. This is confusing.

Reply:
as above